### ISSUE TITLE
[BUG] TF4Rec model serving with Torch script nb is broken

### ISSUE THREAD

**Issue by rnyak** [NVIDIA-Merlin/systems#292]
### Bug description

This [notebook](https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/main/examples/getting-started-session-based/03-serving-session-based-model-torch-backend.ipynb) gives error due to tuple representation (values and lengths) in the config file.

### Steps/Code to reproduce bug

Please run the notebooks in this [folder](https://github.com/NVIDIA-Merlin/Transformers4Rec/tree/main/examples/getting-started-session-based) in order to repro the issue.

### Environment details

- Merlin version:
- Platform:
- Python version:
- PyTorch version (GPU?):
- Tensorflow version (GPU?):

I am using `merlin-pytorch:22.12` image with `23.02` release tag. And same issue occurs if I pull the latest main branches.

**Comment by karlhigley** [comment]
Looks like the error that surfaces here is:
```
InferenceServerException: [StatusCode.INVALID_ARGUMENT] [request id: 1] unexpected shape for input 'category-list__values' for model 'executor_model'. Expected [-1,-1], got [640]
```

**Comment by karlhigley** [comment]
@rnyak Two changes that seem to fix this issue together:
* Apply padding in the NVT workflow using `nvt.ops.ListSlice(-SESSIONS_MAX_LENGTH, pad=True)`
* Insert this snippet after cell 12 in notebook 3, just before printing out the input schema:
```
for col_name, col_schema in input_schema.column_schemas.items():
    input_schema[col_name] = input_schema[col_name].with_shape((None, sparse_max[col_name]))
```

(Those two changes are a workaround to make everything line up while we sort out shapes and ragged support in T4R, but should be okay for now AFAIK)
